bitnami/mysql/Chart.yaml

```diff
@@ -25,4 +25,8 @@ name: mysql
 sources:
   - https://github.com/bitnami/bitnami-docker-mysql
   - https://mysql.com
+<<<<<<< support-overriding-mysql-init-db-configmap
+version: 8.8.29
+=======
 version: 8.8.30
+>>>>>>> master
```

Comment:
Hi,
  Could you take a look to this ? I think you had some changes pending .

Reply:
Sorry, I fixed it.